tutorials/artifactory-gcs-setup/index.md

```diff
@@ -0,0 +1,164 @@
+interoperability storage keys.
+
+Add [this example XML](https://github.com/GoogleCloudPlatform/community/blob/master/tutorials/artifactory-gcs-setup/binarystore-example.xml)
+into the `binarystore.xml` file to setup a default chain that uses
+**`google-storage`** as the binary provider:
+
+You should use the interoperability values from your Cloud Storage account
+(as described in Step 1) for __identity__ and __credential__.  The name of the
+bucket you created in Cloud Storage should be used as the __bucketName__.
+
+#### Other parameters
+
+The full set of parameters available for the **`google-storage`** binary
+provider offer other options that may be useful.
+
+Artifactory uses the JetS3t framework to access Cloud Storage. Some of the
+parameters below are used to set the corresponding value in the framework. For
+more details, please refer to the [JetS3t Configuration Guide](http://www.jets3t.org/toolkit/configuration.html).
+
+| Parameter | Description |
+| --- | --- |
+| __testConnection__ | Default: true. When true, the Artifactory uploads and downloads a file when starting up to verify that the connection to the cloud storage provider is fully functional. |
+| __multiPartLimit__ | Default: 100,000,000 bytes. File size threshold over which file uploads are chunked and multi-threaded. |
+| __identity__ | Your cloud storage provider identity. |
+| __credential__ | Your cloud storage provider authentication credential. |
+| __bucketName__ | Your globally unique bucket name on Cloud Storage. |
+| __path__ | The relative path to your files within the bucket |
+| __proxyIdentity__ __proxyCredential__ __proxyPort__ __proxyHost__ | Corresponding parameters if you are accessing the cloud storage provider through a proxy server. |
+| __port__ | Default: 80. The port number through which you want to access Cloud Storage. You should only use the default value unless you need to contact a different endpoint for testing purposes. |
+| __endpoint__ | Default: commondatastorage.googleapis.com. The Cloud Storage hostname. You should only use the default value unless you need to contact a different endpoint for testing purposes. |
+| __httpsOnly__ | Default: false. Set to true if you only want to access Cloud Storage through a secure https connection. |
+| __httpsPort__ | Default: 443. The port number through which you want to access Cloud Storage securely through https. You should only use the default value unless you need to contact a different endpoint for testing purposes. |
+| __bucketExists__ | Default: false. Only available on _google-storage_. When true, it indicates to the binary provider that a bucket already exists in Google Cloud Storage and therefore does not need to be created. |
+
+For more information, see the Artifactory User's Guide topic [Configuring the Filestore](/confluence/display/RTF5X/Configuring+the+Filestore#ConfiguringtheFilestore-GoogleStorage,S3andS3OldBinaryProviders).
```

Comment:
Hi @jfrogtw, can you fix this link? It's currently a relative link and doesn't work.

Reply:
Sorry, not sure how that happened. It's fixed now.

Reply:
1. Go [here](https://github.com/jfrogtw/community/edit/master/tutorials/artifactory-gcs-setup/index.md?pr=%2FGoogleCloudPlatform%2Fcommunity%2Fpull%2F488)
2. Scroll down to where the relative link is
3. Fix the relative link
4. Add a commit message at the bottom
5. Click "Commit changes"
6. Comment back here that you fixed the link
7. I will merge your PR.

Reply:
How strange, I did that once already, not sure why it didn't take. Made the fix again, it appears to be okay now.